go.mod

```diff
@@ -121,7 +121,7 @@ require (
 	github.com/safchain/ethtool v0.0.0-20200218184317-f459e2d13664
 	github.com/samuel/go-zookeeper v0.0.0-20180130194729-c4fab1ac1bec // indirect
 	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b // indirect
-	github.com/shirou/gopsutil v2.20.9+incompatible
+	github.com/shirou/gopsutil v3.21.1+incompatible
```

Comment:
this might have impacts on other plugins, as I think it's pretty widely used. is this a requirement for this plugin or did it get updated accidentally?

Reply:
looks like the versions within the package have separate paths, so might not be as big a deal to upgrade it, thought we might want to address it in a separate PR.

Reply:
I had to do this because of this issue. https://github.com/shirou/gopsutil/issues/986 It looks like some recent change on golang.org/x/sys broke it.
I couldn't figure out any other solution.

Reply:
I also tried to pin the version of the `golang.org/x/sys` but it is updated to the most recent version whenever I run the `make`.

Do you have any idea?

Reply:
I think I fixed it. Probably I mistakenly modified something in `go.sum`. I checked out the `go.sum` from the upstream and tried it again and it manages to build with the `github.com/shirou/gopsutil v2.20.9+incompatible` and `golang.org/x/sys v0.0.0-20200930185726-fdedc70b468f`.

Reply:
It is interesting because I didn't know the content of the `go.sum` can cause the change of `go.mod`. I though it is unidirectional not bidirectional.